docs/unshareTargets.md

```diff
@@ -0,0 +1,396 @@
+}
+
+input EditTargetInput {
+  sidereal:    EditSiderealInput    # See existing API
+  nonSidereal: EditNonSiderealInput # See existing API
+  # ToO
+}
+```
+
+Each edit in the `editList` list would define exactly one of the `add` / `delete` / `edit` options 
+above.
+
+A few examples might help to clarify the idea.
+
+### Delete the target list altogether 
+
+```json
+{
+  "editObservation": {
+    "observationId": "o-3",
+    "targets": {
+      "science": {
+        "replaceList": []
+      }
+    }
+  }
+}
+```
+
+### Replace the target list entirely with a single target
+
+```json
+{
+  "editObservation": {
+    "observationId": "o-3",
+    "targets": {
+      "science": {
+        "replaceList": [
+          {
+            "sidereal": {
+              "name": "NGC 5949",
+              "ra":   { "hms": "15:28:00.668" },
+              "dec":  { "dms": "64:45:47.4"   },
+              "magnitudes": [{
+                "band":   "B",
+                "value":  12.7,
+                "system": "VEGA"
+              }]
+            }
+          }
+        ]
+      }
+    } 
+  }
+}
+```
+
+### Edit the radial velocity of one of the targets
+
+```json
+{
+  "editObservation": {
+    "observationId": "o-3",
+    "targets": {
+      "science": {
+        "editList": [
+          {
+            "edit": {
+              "sidereal": {
+                "name": "NGC 5949",
+                "radialVelocity": { "metersPerSecond": 423607 }
+              }
+            }
+          }
+        ]
+      }
+    }
+  }
+}
+```
+   
+### Delete one target, add another, leave the rest (if any) untouched
+
+```json
+{
+  "editObservation": {
+    "observationId": "o-3",
+    "targets": {
+      "science": {
+        "editList": [
+          {
+            "delete": "NGC 5949"
+          },
+          {
+            "add": {
+              "sidereal": {
+                "name": "NGC 3269",
+                "ra":  { "hms": "10:29:57.070" },
+                "dec": { "dms": "-35:13:27.8"  },
+                "radialVelocity": { "metersPerSecond": 3753885 },
+                "magnitudes": [{
+                  "band":   "B",
+                  "value":  13.24,
+                  "system": "VEGA"
+                }]
+              }
+            }
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+Because these updates apply to the observation itself, they can seem rather lengthy. The path to 
+arrive at a science target descends through multiple levels. Additional purposed mutations that 
+drill down directly to the level of interest could be added as necessary.  For example, an 
+`editSiderealScienceTarget` mutation could take a program ID, or a list of observation IDs, along 
+with a sidereal target name.  It would then apply provided updates directly.
+
+For example, set the radial velocity of all "NGC 5949" targets in program `p-34d5`.
+
+```json
+{
+  "editSiderealScienceTarget": {
+    "programId":      "p-34d5",
+    "name":           "NGC 5949",
```

Comment:
We'd definitely have the same target name multiple times for different observations. And above, it notes that the name would have to be unique for all the targets within an observation. But, in order to prevent a problem with different observations having the targets with the same name but not otherwise being equal, I think this mutation might need to take a list of observation ids.

Reply:
Ah. You mentioned an optional list of observation ids above. That seems sufficient. It would be up to the user (api user or UI) to know if it made sense to do it for all targets of that name in the program.